.ci/generate-extra-site-links.sh

```diff
@@ -59,8 +59,9 @@ do
   # Read the file from the earliest change to the top. It would read first row 90, then 89, 88..1.
   done < <(head -n "$EARLIEST_CHANGE_LINE_NUMBER" "$CURRENT_XDOC_PATH" | tac)
 
-  # Extract file name from path, i.e. 'config_misc'.
-  CURRENT_XDOC_NAME=$(echo "$CURRENT_XDOC_PATH" | sed 's/src\/xdocs\/\(.*\)\.xml/\1/')
+  # Extract file name from path, i.e. 'config_misc' and remove '.vm' if it exists.
+  CURRENT_XDOC_NAME=$(echo "$CURRENT_XDOC_PATH" | sed 's/src\/xdocs\/\(.*\)\.xml/\1/' |
+    sed 's/.vm//')
```

Comment:
Please keep `|` as leading symbol. It more explicit line wrap and easy to read.
As there might be continuation by `&&`. With lead symbol you do not need to look back.

Same as we do in java with operations in expressions `&&` `||`.

Reply:
Noted. Done.